docs/main/04-guides/02-installing-gitops-agent/01-flux.mdx

```diff
@@ -90,6 +90,13 @@ resource depending on the type of state store you are using.
     authentication opitions for the `GitRepository` resource, see the [FluxCD
     docs](https://fluxcd.io/flux/components/source/gitrepositories/).
 
+    :::info
+    This example depends on a secret called `git-credentials` existing in the
```

Comment:
The example below creates an `ssh-credentials` secret, but the flux document references a `git-credentials` secret. I think this slipped prior reviews, so we should probably make sure that we either:

1. Create the secret and use it, which may change this info box slightly, to something like "the example below **creates** a secret..."
2. Remove the secret entirely and leave the info box as per this PR.

I'd be inclined to update it to (1). wdyt?

Reply:
The secret it creates isn't valid though, and it can't be as it needs the users SSH creds. So I think what you're suggesting is both keep the call out in this PR and update the secret name in this example?

Reply:
yeah, i just mean the name should be the same there; i think the intention of this doc was to say

* create this GitRepository that has a reference to this secret `git-credentials`
* create the secret with the same name (but it's not the same on the example)